daffodil-test/src/test/resources/org/apache/daffodil/section23/dfdl_functions/Functions.tdml

```diff
@@ -466,6 +480,20 @@
         </xs:sequence>
       </xs:complexType>
     </xs:element>
+
+    <xs:element name="contentLength3">
+      <xs:complexType>
+        <xs:sequence>
+          <xs:element name="x" dfdl:lengthKind="explicit" dfdl:length="{ 1 }">
+            <xs:complexType>
+              <xs:sequence>
+                <xs:element name="y" type="xs:int" dfdl:occursCountKind="expression" dfdl:occursCount='{ if (dfdl:contentLength(.., "bits") eq 0) then 1 else 2 }' maxOccurs="2" />
```

Comment:
Hard to code review because lines are too long in this file. When changes to a file are more than a screenful, you have to scroll down to find the horizontal scroll bar, which makes this very challenging.

Reply:
Always forget about that, I've pushed a fixup commit to fix these.